scene/gui/tab_bar.cpp

```diff
@@ -1409,7 +1412,8 @@ Variant TabBar::_handle_get_drag_data(const String &p_type, const Point2 &p_poin
 	set_drag_preview(drag_preview);
 
 	Dictionary drag_data;
-	drag_data["type"] = p_type;
+	drag_data["type"] = "tab";
+	drag_data["tab_type"] = p_type;
```

Comment:
Note that changing this means that some users may need to update it as well, see https://github.com/gilzoide/godot-dockable-container/pull/24
Maybe instead we could keep the type the same and add a `drag_data["is_rearrange_tab"] = true` or something.

Reply:
idk, consistent type makes more sense.
Technically this is not part of the API, so compatibility is not a concern here.